src/conn_req.c

```diff
@@ -154,6 +154,7 @@ rpma_conn_req_accept(struct rpma_conn_req *req,
 	if (ret)
 		goto err_conn_delete;
 
+	rpma_private_data_discard(&req->data);
```

Comment:
wouldn't it make more sense to "move" the private data pointer instead of a) allocating a buffer, b) copying new data into it, c) discarding old buffer?